server/src/main/java/keywhiz/service/daos/SecretSeriesMapper.java

```diff
@@ -50,6 +60,25 @@ public SecretSeries map(SecretsRecord r) {
         r.getCurrent());
   }
 
+  private String getOwner(SecretsRecord r) {
```

Comment:
Might be worth being specific here (and in general in this PR) that this is returning the owner name, and NOT the owner id. So renaming the method to`getOwnerName` would save some mental processing for the reader and clarify that we're getting the group name from the secrets record, which contains an owner id.

Reply:
Great suggestion, I'll see what I can do to distinguish the two in the code.